src/content/en/tools/workbox/v3/reference-docs/latest/module-workbox-webpack-plugin-WorkboxWebpackPlugin.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
```

Comment:
The reference docs are auto-generated, right? Do we need to commit this to the git repo?

cc @petele

Reply:
yes otherwise it won't be added to DevSite. It's currently being done for the current workbox docs.